src/Features/LanguageServer/Protocol/Handler/ExportLspMethodAttribute.cs

```diff
@@ -15,9 +17,11 @@ internal class ExportLspMethodAttribute : ExportAttribute, IRequestHandlerMetada
     {
         public string MethodName { get; }
 
-        public string LanguageName { get; }
+        public string? LanguageName { get; }
+
+        public bool MutatesSolutionState { get; }
```

Comment:
doc

Reply:
even  <see cref=IRequestHandlerMetadata.MutatesSolutionState>

Reply:
This is in a follow up PR